validation/views.py

```diff
@@ -136,7 +136,9 @@ def search_phrases(request):
 
     query = request.GET.get("query")
     all_matches = Phrase.objects.filter(
-        Q(transcription__contains=query) | Q(translation__contains=query)
+        Q(transcription__contains=query)
+        | Q(fuzzy_transcription__contains=query)
```

Comment:
Whenever you're querying the `fuzzy_transcription` field, use `to_indexable_form()`. Here's an example of its usage: https://github.com/UAlbertaALTLab/recording-validation-interface/blob/7aa44c0a3bfd18863841645ded4fd95598de284b/validation/views.py#L348

```suggestion
        | Q(fuzzy_transcription__contains=to_indexable_form(query))
```

This takes care of the "must have all accents or no accents" problem. Now you can have any amount of accents because `to_indexable_form()` is intended to match things with entries in `fuzzy_transcription`

Reply:
Awesome, thank you!